data/json/item_groups.json

```diff
@@ -100,6 +100,7 @@
       [ "pine_bough", 20 ],
       [ "acorns", 15 ],
       [ "pinecone", 5 ],
+      [ "juniper", 1 ],
```

Comment:
I'd suggest making juniper berries more common in the forest, to represent finding little juniper bushes and things